legacy/src/app/partials/nodes-list.html

```diff
@@ -1357,6 +1358,14 @@ <h4 class="page-header__dropdown-title">Add rack controller</h4>
                 </span>
               </div>
             </td>
+            <td class="p-table__cell" aria-label="Available upgrade">
+              <div
+                data-ng-show="controller.versions.update"
```

Comment:
`data-ng-if` is almost universally the better option. `data-ng-show` still renders the full markup with an added `ng-hide` class that gives it `display: none`.

Reply:
Thanks, I'll fix that up.

Reply:
Done.